pkg/infrastructure/gcp/clusterapi/firewallrules.go

```diff
@@ -5,12 +5,20 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/sirupsen/logrus"
 	"google.golang.org/api/compute/v1"
 
+	gcpconfig "github.com/openshift/installer/pkg/asset/installconfig/gcp"
 	"github.com/openshift/installer/pkg/infrastructure/clusterapi"
 	"github.com/openshift/installer/pkg/types"
 )
 
+const (
+	// GCPFirewallPermission is the role/permission to create or skip the creation of
+	// firewall rules for GCP during a xpn installation.
+	GCPFirewallPermission = "compute.firewalls.create"
```

Comment:
Maybe it doesn't need to be exported or there is a better place for it (like pkg/types/gcp)?